examples/failure-policy/failjobset-action.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: jobset.x-k8s.io/v1alpha2
+kind: JobSet
+metadata:
+  name: failjobset-action-example
+spec:
+  failurePolicy:
```

Comment:
Can you add a brief comment to the `failurePolicy` section of each example describing what behavior the user should expect to see when running the example?